webapp/server/index.js

```diff
@@ -1,24 +1,23 @@
 const express = require('express');
 const key = require('./api');
+
 const app = express();
 const request = require('request');
 
 
 const PORT = process.env.PORT || 5000;
 
-app.use(express.static(__dirname +'./../../'));
+app.use(express.static(`${__dirname}./../../`));
 app.listen(PORT);
 
 app.get('/popular', (req, res) => {
-  request('https://api.themoviedb.org/3/movie/popular?api_key='+key.key+'&language=en-US&page=1', (err, response, body)=>{
-    if(err){
+  request(`https://api.themoviedb.org/3/movie/popular?api_key=${key.key}&language=en-US&page=1`, (err, response, body) => {
+    if (err) {
       res.send(err);
-    }
-    else {
+    } else {
       const data = JSON.parse(body);
       res.send(data);
```

Comment:
res.json(data) will send data back as parsed json, without you having to call JSON.parse